### ISSUE TITLE
@example not working with constructor

### ISSUE THREAD

**Issue by D-Marc1** [jsdoc2md/jsdoc-to-markdown#173]
For some reason the `@example` tag works for every method, except for my constructor. I'm using the current version of jsdoc-to-markdown.

**Comment by dashersw** [comment]
This used to work in 2017 (around April) but now stopped working. It's breaking our documentation :(

**Comment by 75lb** [comment]
Is it a `class` constructor? If so, try moving your doclet from the `constructor` to the `class`. 

So, from this:

```js
class Something {
  /**
   * Docs
   */
  constructor() {}
}
```

... to this:

```js
/**
 * Docs
 */
class Something {
  constructor() {}
}
```

Generally, try to avoid documenting a class constructor - there are existing bugs in the jsdoc project on this subject..

**Comment by dashersw** [comment]
i tried that — it still doesn't work.

**Comment by 75lb** [comment]
Fixed and released in [jsdoc2md v5.0.0](https://github.com/jsdoc2md/jsdoc-to-markdown/releases/tag/v5.0.0).
